From fc512cee665f1e2736a701c9960efce5c093bca3 Mon Sep 17 00:00:00 2001
From: Volker Krause <vkrause@kde.org>
Date: Sat, 14 Apr 2018 18:08:38 +0200
Subject: Remove unused D-Bus dependency

Summary: Makes this much easier to build on Android.

Reviewers: #kde_pim, dvratil

Reviewed By: #kde_pim, dvratil

Tags: #kde_pim

Differential Revision: https://phabricator.kde.org/D12198
---
 CMakeLists.txt     | 2 +-
 src/CMakeLists.txt | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f47888b..87408d9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,7 +33,7 @@ ecm_setup_version(PROJECT VARIABLE_PREFIX KContacts
 )
 
 ########### Find packages ###########
-find_package(Qt5 ${QT_REQUIRED_VERSION} REQUIRED COMPONENTS DBus Gui)
+find_package(Qt5 ${QT_REQUIRED_VERSION} REQUIRED COMPONENTS Gui)
 find_package(KF5 ${KF5_VERSION} REQUIRED COMPONENTS CoreAddons I18n Config Codecs)
 
 ########### Targets ###########
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 819e7fe..e1a2094 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -63,7 +63,6 @@ PUBLIC
     KF5::CoreAddons
 PRIVATE
     Qt5::Gui
-    Qt5::DBus
     KF5::ConfigCore
     KF5::I18n
     KF5::Codecs # for the vcard parser
-- 
cgit v0.11.2

